core/monitoring/user-events/src/main/scala/org/apache/openwhisk/core/monitoring/metrics/EventMessage.scala

```diff
@@ -0,0 +1,182 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.openwhisk.core.monitoring.metrics
+
+import java.util.concurrent.TimeUnit
+
+import spray.json._
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+import scala.util.Try
+
+//Classes in this file are taken from OpenWhisk code base
+
+trait EventMessageBody {
+  def typeName: String
+}
+
+object EventMessageBody extends DefaultJsonProtocol {
```

Comment:
As discussed refer to `EventMessage` from common module and move any changes to that file